.github/workflows/docusaurus.yml

```diff
@@ -22,9 +22,9 @@ jobs:
           node-version: 16
 
       - name: Install dependencies
-        run: npm ci
+        run: cd docs && npm ci
       - name: Build website
-        run: npm run build
+        run: cd docs && npm run build
```

Comment:
We might need to run this through some tests, somehow, before merging.
My concern is that you are `cd`ing into docs in the Install Dependencies step and then cd'ing in again here.
Does Github actions preserve current directory in between steps?

Reply:
I ran the tests in my repo and it worked https://github.com/aabedraba/kusk-gateway-docs-2